CHANGELOG.md

```diff
@@ -7,7 +7,11 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Added
 
 - `agate/Button` prop `iconOnly`, `iconPosition` and `minWidth`
+- `agate/DatePicker` prop `onSpotlightDisappear` and `spotlightDisabled`
```

Comment:
I'm wondering should we add these props to `DateTimePicker` too?

Reply:
good idea